internal/pkg/cli/job_init.go

```diff
@@ -252,7 +261,7 @@ func (o *initJobOpts) askJobName() error {
 }
 
 func (o *initJobOpts) askDockerfile() error {
```

Comment:
I think in our selector it would be super nice to provide an option like this, what do you think?
```bash
Which Dockerfile would you like to use for results?  [Use arrows to move, type to filter, ? for more help]
  > ./Dockerfile
  > Use an existing image instead

What's the name of the image to use? <sample ECR url> 
```
Once they select these options the collapsed version would be:
```bash
Dockerfile: Use an existing image instead
Image: <ECR repo URL>
```

Reply:
oooh that looks pretty good!